src/nft/components/bag/BagFooter.tsx

```diff
@@ -172,24 +190,55 @@ export const BagFooter = ({
   return (
     <FooterContainer>
       <Footer>
-        <FooterHeader gap="xs" warningText={!!warningText}>
-          <Row justify="space-between">
-            <div>
-              <ThemedText.HeadlineSmall>Total</ThemedText.HeadlineSmall>
-            </div>
-            <div>
-              <ThemedText.HeadlineSmall>
+        {shouldUsePayWithAnyToken && (
+          <CurrencyRow>
+            <Column gap="xs">
+              <ThemedText.SubHeaderSmall textAlign="start">
+                <Trans>Pay with</Trans>
+              </ThemedText.SubHeaderSmall>
+              <CurrencyInput>
+                <CurrencyLogo currency={inputCurrency} size="24px" />
+                <ThemedText.HeadlineSmall fontWeight={500} lineHeight="24px">
+                  {inputCurrency?.symbol}
```

Comment:
Should the ETH price be aligned with the icon?
![Screen Shot 2023-01-13 at 09 43 04 ](https://user-images.githubusercontent.com/11512321/212385994-69d1f83f-7a2f-4683-bde7-accbed0aac5c.png)

Reply:
actually no - this is the way it is in figma